src/components/CookieConsentBanner.tsx

```diff
@@ -3,12 +3,15 @@
 import { ThemeButton } from "./ThemeButton";
 import { Check, X } from "@phosphor-icons/react";
 import { useCookieContext } from "@/context/CookieContext";
-import { useEffect, useState } from "react";
+import { useEffect, useState, useRef } from "react";
```

Comment:
It looks like you have a couple of small conflicts that need to be resolved with main.